UIUX Update

SSwiniarski opened the pull request:
<!--
👋 Hi, thanks for submitting a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description


This was one of my requests on the last mass UIUX update, I deferred it so we could merge. This is breaking down the overlong "bias in user research" entry into the concept and a number of terms.

<!-- Please write a summary of the change, such as which topic(s) and file(s) that you have edited or created. Please also include relevant motivation and context: -->

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

<!-- Please check ALL the boxes: -->

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

ishg-153 commented on content/uiux/concepts/bias-in-user-research/terms/interview-bias/interview-bias.md:
```diff
@@ -0,0 +1,20 @@
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Interview bias** refers to bias due to a participant's perception of the interviewer. This may be due to a user's own biased perception of an interviewer's ethnicity, age, gender, or physical appearance &mdash; or even details like body language or facial expressions. Interview bias can be difficult to avoid, so it is just important to keep in mind when evaluating the data.
```
I think this entry is supposed to be for Interviewer Bias.

ishg-153 reviewed (changes requested):
Hey @SSwiniarski. I have suggested some changes, please let me know if I misunderstood anything.

For the framing effect entry, can sub it for cognitive bias instead? Since the framing effect stems from Cognitive bias.

SSwiniarski commented:
> Hey @SSwiniarski. I have suggested some changes, please let me know if I misunderstood anything.
> 
> For the framing effect entry, can we sub it for cognitive bias instead? Since the framing effect stems from Cognitive bias.

@ishg-153  I'd rather cognitive bias be done in it's own PR. Other than that, I addressed your comments.

SSwiniarski commented:
@dakshdeepHERE, committed your suggestions.

dakshdeepHERE reviewed (approved):
Looks good
